contrib/create-waku/src/cli.ts

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env node
+
+import fs from 'node:fs'
+import path from 'node:path'
+import prompts from 'prompts'
+import { spawnSync } from 'node:child_process'
+import { red, green, bold } from 'kolorist'
+
+function isValidPackageName(projectName: string) {
+  return /^(?:@[a-z0-9-*~][a-z0-9-*._~]*\/)?[a-z0-9-~][a-z0-9-._~]*$/.test(
+    projectName
+  )
+}
+
+function toValidPackageName(projectName: string) {
+  return projectName
+    .trim()
+    .toLowerCase()
+    .replace(/\s+/g, '-')
+    .replace(/^[._]/, '')
+    .replace(/[^a-z0-9-~]+/g, '-')
+}
+
+// if the dir is empty or not exist
+function canSafelyOverwrite(dir: string) {
+  return !fs.existsSync(dir) || fs.readdirSync(dir).length === 0
+}
+
+async function init() {
+  const cwd = process.cwd()
+
+  let targetDir = '';
+  let defaultProjectName = 'waku-project'
+
+  const CHOICES = fs.readdirSync(path.resolve(cwd, 'template'))
```

Comment:
I'm curious if we can simplify this, because cwd is always `process.cwd()`.

```suggestion
  const CHOICES = fs.readdirSync('template')
```